READMEV2.md

```diff
@@ -0,0 +1,76 @@
+# VEDA Features API
```

Comment:
reviewer note: the reason I have a READMEV2.md here is so that all changes are additive and clearer to reviewers and so we can clean up CDK infra . A followup PR will swap things out and remove CDK refs